Binary_Tree_Top_View/Binary_Tree_Top_View.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+#include <queue>
+#include <map>
+using namespace std;
+
+// Structure of binary tree left, head, right
+struct Node
+{
+    Node *left;
+    Node *right;
+    int head;
+    int data;
+};
+
+// function to create a new node
+Node *newNode(int key)
+{
+    Node *node = new Node();
+    node->left = node->right = NULL;
+    node->data = key;
+    return node;
+}
+
+// function should print the topView of
+// the binary tree
+void topview(Node *root)
+{
+    if (root == NULL)
+        return;
+    queue<Node *> q;
+    map<int, int> m;
```

Comment:
Use better variable name.

Reply:
a queue is used as q in programming. So I changed the map to mp.